scripts/ci-e2e.sh

```diff
@@ -40,11 +40,6 @@ create_cluster() {
 
     if [[ "${REGISTRY}" =~ \.azurecr\.io ]]; then
       az acr login --name "${REGISTRY}"
-      echo "Granting AcrPull permission to the cluster's managed identity"
```

Comment:
we have anonymous pull enabled for the upstream registry so we don't need to grant AcrPull permission.